### PR TITLE
WIP; contract address caching

### PR DESCRIPTION
**Pull request by Anze1m**
# Checklist:

- [ ] Code follows the style guidelines
- [ ] Documentation updated
- [ ] Corner cases evaluated and handled
- [ ] index.js exports updated
- [ ] Automatic tests added/updated
- [ ] Automatic tests pass
- [ ] Sanity check passed
- [ ] Self-review passed

### CODE REVIEW

File: test/wrappers/head_wrapper.js [hunk 1]
```diff
@@ -19,40 +19,40 @@ const {expect} = chai;
...
   let web3;
   let ownerAddress;
-  let head;
   let context;
   let catalogue;
   let storageCatalogue;
   let headWrapper;
   const exampleAddress = '0x97E12BD75bdee72d4975D6df410D2d145b3d8457';
   const deployedMockContracts = {};
+  let addressesForCatalogueConstructor;
```
Comment (vanruch) [inline test/wrappers/head_wrapper.js]: Let's always group 'lets' and 'consts' together
